ui/components/ui/info-tooltip/info-tooltip.js

```diff
@@ -34,7 +35,8 @@ export default function InfoTooltip({
         html={contentText}
         theme={wide ? 'tippy-tooltip-wideInfo' : 'tippy-tooltip-info'}
       >
-        <InfoTooltipIcon fillColor={iconFillColor} />
+        {/* <InfoTooltipIcon fillColor={iconFillColor} /> */}
+        <HelpOutLine color="#222222" />
```

Comment:
```suggestion
        <HelpOutlineIcon color="#222222" />
```

Reply:
fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

Reply:
- Verified